cmd/zoekt-indexserver/main.go

```diff
@@ -262,6 +359,8 @@ func main() {
 	dataDir := flag.String("data_dir",
 		filepath.Join(os.Getenv("HOME"), "zoekt-serving"), "directory holding all data.")
 	indexDir := flag.String("index_dir", "", "directory holding index shards. Defaults to $data_dir/index/")
+	dynamicIndexServer := flag.Bool("dynamic_server", false, "whether or not to start dynamic web server to allow client to specify repos to index dynamically")
```

Comment:
this flag doesn't seem necessary? Rather you can just rely on listen and make the default empty so it doesn't start.

Reply:
Not necessary now that it's a separate binary.